src/components/Navigation/Navigation.scss

```diff
@@ -287,14 +290,15 @@ $secondary-item-font-size: 15px;
 
     @include breakpoint-after(nav-min-window-corrected()) {
       font-size: $item-font-size-small;
-      line-height: 28px;
+      line-height: 1;
+      padding-left: nav(icon-size) + spacing(base-tight);
```

Comment:
**question:** Where is `spacing()` coming from? `shared/_spacing.scss` appears to have been removed in v9.0.0